modules/extensions-support/src/main/java/org/mule/module/extension/internal/util/SingleValueSetter.java

```diff
@@ -45,6 +46,6 @@ public SingleValueSetter(ParameterModel parameterModel, Field field)
     @Override
     public void set(Object target, ResolverSetResult resolverSetResult)
     {
-        fieldSetter.set(target, resolverSetResult.get(parameterModel));
+        Optional.ofNullable(resolverSetResult.get(parameterModel)).ifPresent(v -> fieldSetter.set(target, v));
```

Comment:
Optional makes sense when you are returning nulllable values. In this case, which is completely internal, this becomes overkill. Use an if instead